collection.go

```diff
@@ -170,10 +187,11 @@ func (c *Collection) DiscoverSQLMigrations(dir string) error {
 		return ms[len(ms)-1]
 	}
 
-	files, err := ioutil.ReadDir(dir)
+	files, err := f.Readdir(-1)
 	if err != nil {
 		return err
 	}
+	sort.Slice(files, func(i, j int) bool { return files[i].Name() < files[j].Name() })
```

Comment:
Do we really need this? I think this package already must sort migrations as they are added.